entrypoints/popup/router.tsx

```diff
@@ -184,6 +181,14 @@ export const router = createHashRouter([
                     path: "kas-asset",
                     element: <KasAsset />,
                   },
+                  {
+                    path: "ledger-connect-for-sign",
+                    element: <LedgerConnectForSign />,
+                  },
+                  {
+                    path: "ledger-connect-for-sign-failed",
+                    element: <LedgerConnectFroSignFailed />,
```

Comment:
Typo in the component's name `Fro`

Reply:
https://github.com/forbole/kastle/pull/41/commits/5a20d896907b1c4152846cc4a891daf12694e85f